css/main-title.css

```diff
@@ -31,3 +33,11 @@
   height: 2px;
   background-color: #c88859;
 }
+
+.text-l {
```

Comment:
text-l, text-r, text-c cũng vậy, dùng chung cho toàn bộ các trang. nên đưa nó vào style.css dùng chung luôn.
Khi nào nó dính líu tới main-title mới đặt vào main-title.css.
Ví dụ đoạn code này có dính líu tới main title:
.comp-main-title.text-r { }